src/main.ts

```diff
@@ -11,34 +11,38 @@ const term = new Terminal({
   fontFamily: "Jetbrains Mono",
   theme: {
     background: "rgb(47, 47, 47)",
-  }
+  },
 });
 term.loadAddon(fitAddon);
 term.open(terminalElement);
 
 // Make the terminal fit all the window size
-function fitTerminal(){
+async function fitTerminal() {
   fitAddon.fit();
   void invoke<string>("async_resize_pty", {
     rows: term.rows,
     cols: term.cols,
   });
+  // Create a shell if it doesn't exist
```

Comment:
What's the point of this? I don't get it

Reply:
Sorry that was a mistake on my part. Now, the shell should be created before the  the writeToPty.